scapy/contrib/ppi_geotag.uts

```diff
@@ -31,6 +31,8 @@ assert GPSTime_Field("GPSTime", None).delta == 0.0
 = Define local_to_utc
 
 def local_to_utc(local_time):
+    if six.PY3:
+        return local_time
```

Comment:
Seems that this trick (which is a bug under python 2) was fixed on python 3

Reply:
Can you add a comment in the code rather than in the pull-request?